frontend/amp/components/Header.tsx

```diff
@@ -200,10 +200,11 @@ const pillarLinks = (pillars: PillarType[], activePillar: Pillar) => (
 const supportLink =
     'https://support.theguardian.com/?INTCMP=header_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_support%22%7D';
 
-const Header: React.SFC<{ nav: NavType; activePillar: Pillar }> = ({
-    nav,
-    activePillar,
-}) => (
+const Header: React.SFC<{
+    nav: NavType;
+    activePillar: Pillar;
+    edition: Edition;
```

Comment:
It doesn't look like we're using the `edition` in this component

Reply:
Good spot. Corrected.